apis/nucleus/src/components/listbox/components/ListBoxRowColumn/ListBoxRowColumn.jsx

```diff
@@ -102,7 +102,17 @@ function RowColumn({ index, rowIndex, columnIndex, style, data }) {
   }, [rowRef, focusListItems.first, focusListItems.last]);
 
   const handleKeyDownCallback = useCallback(
-    getRowsKeyboardNavigation({ ...actions, focusListItems, keyboard, isModal }),
+    getRowsKeyboardNavigation({
+      ...actions,
+      focusListItems,
+      keyboard,
+      isModal,
+      rowCount,
+      columnCount,
+      rowIndex,
+      columnIndex,
+      layoutOrder,
+    }),
     [actions, keyboard?.innerTabStops]
```

Comment:
you probably should add the new parameters as dependencies here
(to get an updated callback if they change)